### ISSUE TITLE
Server Down?? OpenPose Demo Windows Unknown Error

### ISSUE THREAD

**Issue by ryshaff** [CMU-Perceptual-Computing-Lab/openpose#2246]
Based on what I read in the previous post: https://github.com/CMU-Perceptual-Computing-Lab/openpose/issues/1955

I am getting the same following error as the post above and from the post above this has led me to believe the server is down and needs to "awakened" in order to download the models properly.

This is the output I get when I follow step 1 of the OpenPose instructions to download the demo. Note I have downloaded the proper Microsoft Visual C++ Redistributable and still get the Unknow Error. 

![image](https://github.com/CMU-Perceptual-Computing-Lab/openpose/assets/90356448/e9313ca1-3095-4b16-9780-5c31e060d374)

I believe the server is down for OpenPose and the models cannot be downloaded!

**Comment by SashaEm** [comment]
Hi, did you manage to solve this issue?

**Comment by PippoApps** [comment]
Still down.

**Comment by AA1HSHH** [comment]
face same problem

**Comment by ryshaff** [comment]
I solved this issue a while ago, it is not too difficult, but for people who don't know anything about Github repositories, this is hard to follow. If you are not experienced with GitHub repositories and want to use Openpose's human pose estimation capabilities, use Opencap software instead: https://www.opencap.ai/.  This application utilizes Openpose's human pose estimation models but is 100X more user-friendly for people who don't know much about coding.  Additionally, Opencap may help exponentially with the human pose estimation task you're trying to complete (3D kinematics and 3D kinetics) so definitely check it out! 

DISCLAIMER: The only reason you have continued to read the post is that you are attempting to get the OpenPose demo to work. The only reason you are testing the OpenPose Demo is that you want to analyze multiple subjects simultaneously for non-biomechanics purposes. If you are attempting to test biomechanics USE OPENCAP, SERIOUSLY!

Now, it has been a while since I fixed this. So here is what I remember I think you need to do to get this application to work. 

1. Download the Openpose Demo
2. See if you still get the error I described above in my original post
3. If the error still occurs go to Post #1602 , You can read the entire post and see what happened, but you will see that Dr. Hidalgo's links no longer work to manually download the models. So retrieve them from this user: zoheezus
<img width="824" alt="User Screenshot" src="https://github.com/CMU-Perceptual-Computing-Lab/openpose/assets/90356448/4d5c7f81-e8d8-4e77-9203-fa8ff2bb9166">

4. After the models are downloaded you have to fill in the missing files from the Demos repository on your computer to match the location of the example OpenPose repository on GitHub. Basically, move the Model files into the correct folders so the Demo application can access them when you run the software. 

5. Make sure your demo application has all the correct files, just like the GitHub repository. Make sure the demo application has all the files that the online repository has. They should almost be identical. The Demo Application will tell you the files that are missing when you try to run it and it does not work. Basically compare your demo files to the blue folders of the online repository:
<img width="934" alt="repositorycomparison" src="https://github.com/CMU-Perceptual-Computing-Lab/openpose/assets/90356448/505f52f5-cb5e-4ec0-9488-a05a1db018e2">

6. Try running the demo application now, I don't think I am missing anything. Hopefully, it works!
